test/geocode-test.js

```diff
@@ -7,24 +7,28 @@ var geocode = require('../lib/geocode');
 vows.describe('Geocode').addBatch({
   'When requesting a valid geocode': {
     topic: function () {
-      geocode.geocode({ text: "920 SW 3rd Ave, Portland, OR 97204" }, this.callback);
+      geocode.geocode({ 
+        text: "920 SW 3rd Ave, Portland, OR 97204"
+      }, this.callback);
     },
     'It should return the correct latitude and longitude': function (err, data) {
       assert.equal(err, null);
-      assert.equal(data.locations.length, 1);
-      assert.equal(data.locations[0].feature.geometry.x.toPrecision(5), (-122.67633658436517).toPrecision(5));
-      assert.equal(data.locations[0].feature.geometry.y.toPrecision(5), (45.5167324388521).toPrecision(5));
+      assert.equal(data.candidates.length, 10);
```

Comment:
`> 0` would be sufficient. my impression is that the test only verifies that the request actually generates results.

if i was more ambitious, i'd rewrite the tests with a mock server and skip verifying a lot of the response altogether.

thanks for all this **good** feedback!